CONTRIBUTING.md

```diff
@@ -73,7 +73,15 @@ source .tox/unit/bin/activate
 ```
 
 #### Manual testing
-Alerts can be created using alertmanager's HTTP API,
+Alerts can be created using [`amtool`](https://manpages.debian.org/testing/prometheus-alertmanager/amtool.1.en.html),
+
+```shell
+amtool alert add alertname=oops service="my-service" severety=warning \
```

Comment:
The word “severety” is severely misspelled :D

Also the example uses addresses that do not sound quite realistic for LMA on K8s.

Reply:
Fixed spelling.
Does it matter what address I give as an example for a manual test? What would you rather have there? Something like `prometheus.juju`?

Reply:
I wish we knew how ingress routes will look like :-(